tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/add-operation-negative.bal

```diff
@@ -112,3 +112,42 @@ function errorFunction4(int|float a, ints|float b) {
     int _ = a + b;  
     int _ = b + b; 
 }
+
+function testAdditiveExprWithUnionOperandBelongingToSingleBasicTypeNegative() {
+    string|("a"|"b") a = "a";
+    _ = a + 1;
+    _ = 1 + a;
+    _ = a + a; // OK
```

Comment:
Why do we add these tests to the `add-operation-negative.bal` file? Shouldn't they be moved to the `add-operation` file?

Reply:
That already has tests with unions, right? This was just an additional validation.